watchman/checks.py

```diff
@@ -67,6 +73,14 @@ def _check_storage():
     return {"ok": True}
 
 
+@check
+def _check_amqp_connection(connection_url):
+    conn_params = pika.URLParameters(connection_url)
+    connection = pika.BlockingConnection(conn_params)
+    connection.close()
+    return {'ok': True}
```

Comment:
Can we use some identifier of the connection similar to what cache and database checks do? A possible candidate would be the `vhost` name.

This file seems to favour double quotes